### DISCUSSION TITLE
UI

### DISCUSSION THREAD

**Discussion by saraivetesigv** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#69] [Q&A]
Buenas tardes, yo tengo un poco de dudas sobre como hacer el ultimo punto de las practicas 5-UI
Si alguien me pudiera orientar, lo agradezco.

**Comment by GabrielaEsquivel** [comment]
¡Hola, Sarai! Para el punto número cinco, en este link https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/blob/main/01%20-%20INTRO/practicas/5.-ui.md el MC ha dejado tres herramientas distintas con las cuales tú podrás, vaya, diseñar cómo se verá tu aplicación.
Estas herramientas (de muchas) te proveen de componentes & tools que puedes utilizar para ir diseñando, es decir, lo que debes hacer es algo parecido a esto:
https://www.sleepydays.es/wp-content/uploads/2016/06/descargar-mockup-gratis-web-app-000.jpg

Aclaro: no es obligatorio que utilices alguna de esas tres opciones, puedes escoger otra que tú conozcas o con la que te sientas cómoda. Con tal de que consigas el producto de la práctica, todo cool 👍

**Reply by saraivetesigv** [reply]
Muchas gracias por la ayuda y por la imagen, creo ya darme una idea de como hacerlo

**Reply by saraivetesigv** [reply]
No pude verlo :)

**Reply by GabrielaEsquivel** [reply]
Ups, dio 404, déjame te paso un ss de un mockup que hice hace tiempo, algo así tendrías que hacer, pero más bonito
https://user-images.githubusercontent.com/99062142/155256759-5c8b61b8-0e58-41ff-b059-26fa848fda70.png

**Reply by GabrielaEsquivel** [reply]
Y más general, yo ahí voy haciendo pasito a pasito porque era para mi tesina jajajajaja, pero solo diseñas las páginas/vistas general, algo muy cercano a tu producto final

**Reply by saraivetesigv** [reply]
Muchas gracias, ya entendi pero eso lo hice en el paso 4, cual es la diferencia con el paso 5
Perdon la molestia de los mensajes

**Reply by GabrielaEsquivel** [reply]
No es molestia, no te preocupes! El paso 4 es el boceto y el paso 5 ya no es un boceto en sí, es decir, ahora sí lo diseñas padre tomando en cuenta los mockups que hiciste en el paso 4

**Reply by GabrielaEsquivel** [reply]
Pongamoslo así, el paso 4 es el esqueleto de un cuerpo y el paso 5 la piel con el esqueleto, algo muy burdo pero creo que sí aplica jajajaj

**Reply by saraivetesigv** [reply]
a bien. perfecto. Entonces asi lo aplicare. Muchas gracias por explicarme.

**Reply by GabrielaEsquivel** [reply]
No hay de qué! <3

**Comment by doguedogue** [comment]
Es el diseño de las pantallas que le vas a mostrar al usuario, un bosquejo, no tan a detalle, pero que da claridad de que vas a hacer.

https://www.lucidchart.com/pages/es/que-es-un-wireframe-para-un-sitio-web

**Reply by saraivetesigv** [reply]
Gracias por ayudarme.

**Comment by guillmh** [comment]
Yo lleve un poco de UX/UI, básicamente lo que tienes que hacer es la evolución de los wireframes, es decir los mockups que es una representación de las pantallas pero mas detalladas ya definiendo colores, gráficos etc.. así es como yo lo entendí espero te sirva.

**Comment by CarlosSalinasRuiz** [comment]
Hola buenos días, a mi parecer es que UX es el boceto ya sea a mano o digital de la estructura que lleva la pagina o aplicación, en donde no hay imágenes, texto o colores acorde los requerimientos. Y sobre la parte UI se enfoca a previo boceto que se hizo en UX en el cual vamos a ir siendo creativos en colocar las imágenes, textos y colores.

Un ejemplo visual de como debería de quedar la practica es la siguiente:
![273820373_1549036282141018_2569304555413662374_n](https://user-images.githubusercontent.com/41224166/155331862-f74a1127-30c6-4e92-8daf-51f9525f1b95.gif)

UX:
![image](https://user-images.githubusercontent.com/41224166/155331962-acb3886d-ac53-4a1c-a3b9-003b60062f2f.png)

UI:
![image](https://user-images.githubusercontent.com/41224166/155331987-ba0b0486-6ae5-4942-acef-987b9d88adc5.png)
